app/models/goldencobra/template.rb

```diff
@@ -0,0 +1,25 @@
+module Goldencobra
+  class Template < ActiveRecord::Base
+    attr_accessible :title, :layout_file_name
+
+    # List all layout files found in folder /app/views/layouts
+    #
```

Comment:
Hätte mir gerne noch gewünscht, dass hier auch steht, dass Dateiendungen abgeschnitten werden und alle Dateien ignoriert werden, die mit `_` beginnen. Das macht das Lesen der Methode um ein Vielfaches einfacher, da man quasi schon weiß was passiert und nicht in den speziellen `.map` und .delete_if` recherchiert werden muss.
Die Variable `a` hilft da auch nicht weiter. Könnte auch `filename` heißen, oder?

Für den Regexp könnte auch https://apidock.com/ruby/REXML/Functions/starts_with/class eingesetzt werden, oder? Wäre nochmal lesbarer an der Stelle.